pickee_api/views.py

```diff
@@ -93,7 +93,18 @@ def user_signup(request):
 
 def user_logout(request):
     logout(request)
-    return redirect('index')
+    return redirect('home')
+
+def profile_main(request):
+    user_profile_data = { "age": request.user.age, "gender": request.user.gender }
+    data = json.dumps({
+        "user": {**get_user_data(request), **user_profile_data}
+    });
+    context = {'data': data}
+    return render(request, 'profile_main.html', context=context)
+
+
+# Helper methods
 
 def get_user_data(request):
```

Comment:
We get user data through the `api/users` endpoint

Reply:
We need user data not from the API, but at the page load